### ISSUE TITLE
Misconfigured file ClassifierRule

### ISSUE THREAD

**Issue by rzdhop** [SnaffCon/Snaffler#123]
Hi, 
I've got an issue, the exe work perfectly but here is the printing : 
![Capture d’écran 2023-06-12 151932](https://github.com/SnaffCon/Snaffler/assets/45741890/b7c4c96e-c015-4adf-a27f-34e759ac0bdb)

I've tried to recompile the exe myself just in case, but still.
I have to say that the scan works, just there are those errors popping and over flooding my screen.

**Comment by l0ss** [comment]
Can you please provide the full set of arguments that you're using to run it? I've just tested the 1.0.112 release with `Snaffler.exe -s -i C:\` and it seems to be working fine with none of the errors you're showing above.

**Comment by rzdhop** [comment]
Yes ofc, (i'm using the latest release too) 
`snaffler.exe  -y  -o 'result.tsv'  -s   -j 150  -d 'domain.local'  -b 2  -c 'domain.local' `
I'm trying to scan all shares accross an entire domain, I don't have the issue without the -b parameter (but i need it, or else too many things are returned)

**Comment by l0ss** [comment]
yeah ok this is a bug in how the `-b` functionality works, i'll try to figure it out when i have some time - in the meantime i suggest running without `-b` and just use good old fashioned grep to filter out the lower triage levels.

**Comment by Zamanry** [comment]
Just ran into this same issue. The command I was using was:
```
& snaffler.exe -s -o "Snaffler.txt" -d domain.tld -c dc.domain.tld -b 3
```
I would offer an exact version, but the help options aren't helpful. It's at least 2x years old.

**Comment by l0ss** [comment]
I'm trying to reproduce this - does this occur immediately when you run Snaffler or does it happen when it hits a certain type of file?

**Comment by Zamanry** [comment]
From my memory, I believe the error appeared when it hit a certain file. I know my teams run into this a lot. We run this against entire domains. So I don’t have the info of which share/host was the issue.
